Controllers/StudentController.js

```diff
@@ -94,15 +94,15 @@ const GetBook = async (req, res) => {
 
   try {
     let book;
-    if(author || title){
+    if (author || title) {
       const query = {};
       if (title) query.title = new RegExp(title, "i");
       if (author) query.author = new RegExp(author, "i");
-        book = await Book.find({
+      book = await Book.find({
```

Comment:
Can you check is there any way to efficiently handle the or case in terms of code?